lib/handlers/healthmonitor.js

```diff
@@ -81,11 +90,59 @@ function initFhHealth() {
     fhhealth.addCriticalTest('Check fh-messaging running', checkFhMessaging);
   } else {
     fhhealth.addCriticalTest('Check fh-ditch status', checkDitch);
+    fhhealth.addCriticalTest('Check apps reachable', checkApps);
   }
 
   return fhhealth;
 }
 
+function checkIfAppsAreReachable(apps, cb) {
+
+}
+
+function getAppsUrl(dynos, cb) {
+
+  var res = {};
+
+  async.each(dynos, function (dyno) {
+    dfc.apps.list(dyno.dyno, 'detailed', function(err, apps) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)